### DISCUSSION TITLE
Vailant: Kühlung aktivieren

### DISCUSSION THREAD

**Discussion by m-ask** [john30/ebusd#756] [Q&A]
### Description

Hallo!
Ich habe folgendes Problem.
Mittlerweile kann ich perfekt die Daten meine Vailant Heizung auslesen und Heizung/Warmwasser steuern.
Was ich leider nicht raus finde, ist wie ich die passive Kühlung über ebusd aktivieren kann.
Manuell über das Bedienteil wird doch die Anforderung auch über den EBUS übertragen?
Kann mir jemand dabei helfen?
Danke
m-ask

**Comment by mwildbolz** [comment]
Vielleicht hilft dir dieser Thread ein bisschen weiter - [https://github.com/john30/ebusd/issues/438](https://github.com/john30/ebusd/issues/438)

**Comment by m-ask** [comment]
Hallo mwildbolz!
Bei meiner Vailant geotherm gibt es für die Kühlung keinen Automatik Modus.
Die Aktivierung geht nur über die Vorgabe wieviel Tage die Kühlung aktiv sein soll.
Meine Hoffnung war das man dieses Vorgabe auch per ebus senden kann.

**Comment by mwildbolz** [comment]
OK, interessant.
Ich hab irgendwann einmal eine Konfiguration auch für die manuelle Kühlung gefunden, vielleicht funktioniert das auch bei dir:
https://github.com/mwildbolz/ebusd-configuration/blob/f071d4d0bf766806a64490176ceaed2d5d046b5f/ebusd-2.1.x/de/vaillant/15.700.csv#L7

Lg

**Comment by m-ask** [comment]
die Konfiguration schaut so aus als ob es genau das wäre was ich brauche
jetzt hätte ich noch eine Frage:
ich wollte mal zum Testen ManualCooling auslesen und hab das über read define und read hex versucht nur bekomme ich da immer empty oder Fehler,  ich hab das dann auch noch mit einer bekannten Größe probiert da kommt das gleiche also gehe ich davon aus, dass ich das Kommando nicht richtig verwende 
kannst du mir evtl. weiterhelfen (hab schon fast alle Varianten probiert)
ebusctl read -def r,test,MCooling,,,10,b524,0600    --> liefert empty (--enabledefine ist aktiviert)
ebusctl read -h 10b524020600 --> liefert in allen möglichen Varianten einen Fehler
Hast du da eine Idee, bzw. wie kann ich ebusd dazu bringen die 15.700.csv zu verwenden
danke

**Comment by m-ask** [comment]
Hallo!
Ich hab jetzt folgende Lösung gefunden.
Wenn man die beiden Felder CoolingDay und CoolingActive auf write stellt, dann kann man damit die Kühlung aktivieren.
Zuerst CoolingDay einen Wert schreiben und dann mit CoolingActive aktivieren. Ob das in der Wärmepumpe ein Problem macht weiß ich nicht.
Unter quick.inc hätte ich die gleichen Befehle wie es auch bei der originalen Steuerung drinen sind gefunden. Nur fehlt hier leider die Kühlung. Weiß jemand wie hier die Erweiterung ausschauen würde.

**Reply by dieterguenter** [reply]
Hallo m_ask,
bei meiner Wärmepumpe kann ich die Kühlung auch mit Anfangs- und End-Datum aktivieren (zumindest übers VRC720). Und offensichtlich muss man auch eine Status-Info auf Kühlern stellen. 
In der 15.700.csv ist ja nur die Anzahl der Tage enthalten. Gilt das dann ab "heute" oder hast Du eine Möglichkeit gefunden, auch das Anfangsdatum, wird bei dir wohl CoolingDay sein, vorzugeben?
Wäre super, wenn Du mir dazu die Zeilen für die CSV-Datei mitteilen könntest. Danke.
Gruß Dieter

**Comment by mwildbolz** [comment]
Ich kann dir momentan leider nicht weiterhelfen, da mein ebus-Koppler defekt ist :-(
Aber soweit ich mich erinnern kann, hat das bei mir allein durch Setzen der Variable "ManualCooling" funktioniert - allerdings unter Berücksichtigung der Ein-/Ausschaltbedingungen der Kühlung (mittlere Außentemperatur, etc.)

Lg

**Comment by mf76130** [comment]
Hallo,
Kann ich jetzt einfach einen MQTT Befehl senden dafür?
ebusd/700/CoolingDay/set mit Payload 1
ebusd/700/CoolingActive/set mit Payload 1

oder müssen die anders ausschauen?

mfG
Michael

**Comment by dieterguenter** [comment]
Wenn beim ersten MQTT-Befehl ein CoolingDay eingetragen werden muss, wirst Du mit einer einfachen 1 wahrscheinlich nicht weiterkommen. 
Bei mir ist noch ein OpModeCooling in der csv-Datei. Ich gehe mal davon aus, dass das auch noch auf "Auto" gesetzt werden muss.
Aber letztendlich kannst Du es ja ausprobieren. An der Vorlauftemperatur müsste man ja erkennen ob es funktioniert.

MfG Dieter


Am 17. Juni 2024 10:45:44 MESZ schrieb mf76130 ***@***.***>:

**Comment by mf76130** [comment]
Nö, hat nicht so einfach funktioniert :-)

**Reply by mzwanzi** [reply]
Hallo,

kämpfe mit dem gleichem Thema, hast du es dann noch hin bekommen?

lg M

**Reply by mf76130** [reply]
nein noch nicht :-)
Ich habs manuell so eingestellt das es nur unter Tags kühlt (PV) und manuel kühlen für 7 Tage.
Wenn ich mehr Zeit hab, spiele ich mich vielleicht weiter.

**Reply by mzwanzi** [reply]
ah ich hab genau den gleichen Anwendungsfall :-)

hab das jetzt noch bisschen probiert.
Sobald ich über ein MQTT:Publish das Topic ebusd/700/ManualCooling/set auf z.b. 2 setze, geht er aufs manuelle Kühlen.
Beim setzen auf 0 wird das manuelle kühlen abgebrochen.

Hast du das unter Tags kühlen über den Zeitplan gemacht? Nimmt er den auch fürs manuelle kühlen? Werde da nicht ganz schlau aus der Anleitung

**Reply by mf76130** [reply]
Hi,
Ich hab einen Zeitplan erstellt fürs kühlen, jeden Tag von 8:00 - 19:00 ist es erlaubt.
Er scheint das auch brav fürs manuelle Kühlen zu nehmen.
Hab das heute auch probiert MQTT:Publish das Topic ebusd/700/ManualCooling/set 2, ändert bei mir nichts, ausser das ich das halt im MQTT Explorer sehe.
Wo siehst Du das der aufs manuelle Kühlen stellt?

**Reply by mzwanzi** [reply]
ich werd da noch nicht ganz schlau draus, muss mir das mal auf längere Zeit ansehen.

Ich sehe das bei mir am Bedienteil VR700, im MQTT kommt da nämlich bei mir auch nichts zurück.
Ich werde das mal die Tage beobachten wie sich das ganze verhält, sollte man ja an der Vorlauftemperatur gut erkennen :-)

**Reply by mf76130** [reply]
am Wärmepumpenstatus sollte man es auch sehen:
![image](https://github.com/john30/ebusd/assets/653889/fc9f6199-2fae-4fce-b68a-c5e596c6c7b4)

Ich hab mir jetzt mal die 15.700.csv geladen, die den "OpModeCooling" Eintrag drinnen hat.

MQTT config für Wärmepumpe Status
  - name: "Wärmepumpe Status"
    state_topic: "ebusd/hmu/State"
    value_template: "{{ value_json['3'].value }}"
    json_attributes_topic: "ebusd/hmu/State"
    json_attributes_template: "{{ value_json | tojson }}"

**Reply by mf76130** [reply]
und ich kann bestätigen jetzt gehts mit ebusd/700/ManualCooling/set "2" - dann wird die Anlage auf manuelles kühlen für 2 Tage gestellt. Ist auch am Display sichtbar und kühlt auch nur in dem eingestellten Zeitfenster.
das 2 kann durch jede Zahl ersetzt werden und ist dann die Tage der manuellen Kühlung.

und mit 0 wird es ausgeschalten.

**Reply by mzwanzi** [reply]
bei mir zeigt er jetzt auch alles richtig an, mittlerweile hab ich auch ManualCooling vom ebus koppler geliefert bekommen.
Spannenderweise bekomm ich z.b. die Luftfeuchtigkeit gar nicht.

Aber das war ja jetzt dann noch einfacher als anfangs gedacht :-)

**Comment by dieterguenter** [comment]
Hi, kannst Du mir bitte mitteilen, in welcher CSV-Datei der Eintrag fürs ManualCooling drin ist? Das mit der Vorgabe von Tagen wäre ja genial einfach.
Gruß und Dank.

Am 20. Juni 2024 10:30:42 MESZ schrieb mf76130 ***@***.***>:

**Reply by mf76130** [reply]
https://github.com/john30/ebusd-configuration/blob/master/ebusd-2.1.x/de/vaillant/15.700.csv

**Reply by mf76130** [reply]
Die hab ich gefunden, wie ich nach deinem Eintrag "OpModeCooling" gesucht habe.

**Reply by mf76130** [reply]
man muss auch dazusagen, dass die Wärmepumpe wenn sie mal zu kühlen angefangen hat sich natürlich nicht einfach so wieder ausschalten lässt!

**Reply by mzwanzi** [reply]
Ich werde auch nicht so ganz schlau drauß wie sie wann kühlt?
Bei mir hat sie scheinbar heute morgen mal kurz die Vorlauftemperatur gesenkt  für genau 10 min, jetzt ging sie wieder hinauf, seitdem ist nichts passiert. 
Ich hab da ehrlicherweise noch keine Erfahrung, die WP ist erst im Oktober montiert worden, daher fehlt mir da noch etwas die Erfahrung :-)

**Reply by mf76130** [reply]
Also bei Vailant geht das so:
Bei Vollmond musst Du dein erst geborenes Opfern, dabei muss es gerade über 30° C haben, dann geht den ganzen Sommer die Kühlung :-)

Nein, natürlich nicht, es ist noch blöder: Es gibt da die Temperatur Kühlung "AT Kühlen starten", wenn die überschritten ist für 2 oder 3 Tage und die Temperatur "AT Kühlen beenden" nicht unterschritten wird, dann kühlts.
Wennst halt beimKühlen beenden 18° stehen hast und in der Nacht gibts 16°, fangt der Tagescounter wieder von vorne an - saudämlich.
![2024-06-20 12 50 38](https://github.com/john30/ebusd/assets/653889/bed2098b-cd9c-46be-8533-bc7b81b61db4)
![2024-06-20 12 51 24](https://github.com/john30/ebusd/assets/653889/a436092a-556c-4ec9-84d9-2ed396e77b40)

**Reply by mwildbolz** [reply]
Ein bisschen genauer kann ich das durch meine Erfahrungswerte noch spezifizieren. Kurz zusammengefasst:
- **durchschn. Außentemperatur** muss > "AT Kühlen starten" sein
- _IST Außentemperatur_ muss > "AT Kühlen starten" sein

dann startet die WP mit dem Kühlen.
Fällt die _IST-Außentemperatur_ unter "AT Kühlen beenden", wird die Kühlung beendet. Startet aber wieder, sobald die oben genannten 2 Parameter wieder erfüllt sind.

Von 2-3 Tagen warten kann ich nicht berichten, ich konnte das Verhalten auf die angegebenen Schwellwerte zurückführen. Bei mir übrigens 23°C und 17°C

Meine WP: VWL117/4

Wichtiger Hinweis noch: Wenn die WP aus/eingeschaltet wird, wird der Mittelwert der Außentemperatur zurückgesetzt - dann kann es u.U. wieder länger dauern, bis der Schwellwert überschritten wird...

Lg

**Reply by mzwanzi** [reply]
Vielen Dank für eure Antworten. 

Fürs automatische kühlen hört sich das ja eh "logisch" an. Aber wozu gibts dann das manuelle kühlen? 

Bei mir war vom Installateur auf jeden Fall noch 20°C für AT Kühlen beenden gesetzt, hab das irgendwie komplett überlesen, also vielen Dank für die Fotos, hab das jetzt mal auf 16°C gesetzt und werd das jetzt mal die nächsten Tage beobachten :-)

**Comment by dieterguenter** [comment]
Weiß irgendjemand wie das Äquivalent zu ManualCooling aus der 15.700.csv bei der 15.ctlv2.csv heisst oder heißen könnte? Vaillant wird doch nicht von Wärmepumpe zu Wärmepumpe das Interface ändern.

Am 20. Juni 2024 20:27:28 MESZ schrieb mzwanzi ***@***.***>:

**Reply by mzwanzi** [reply]
hast du versucht das manualcooling z.b. auf 4 tage zu setzten und dann mit MQTT Explorer nach "4" zu suchen? evtl. bekommst du so raus ob das wo gesetzt wird. Aber ich traue ihnen auch zu das von Type zu Type unterschiedlich zu behandeln...

**Reply by mf76130** [reply]
Es scheint so als hätte ich mit der neuen csv ebusd_700_prenergysumhwc_energy4 verloren.
Obwohl die Inhalte bezüglich diesem Wert identisch sind in den CSV, vielleicht dauert es einfach länger bis das upgedatet wird.

Edit: Hat sich wieder beruhigt und alle Werte da.

**Reply by Dette70** [reply]
Gibt es dazu schon Informationen: "Kühlung über mehrere Tage" via 15.ctlv2.csv?

Ich habe den Datenpunkt 1:1 aus der 15.700.csv kopiert, allerdings kann er die Variable (CoolingManual) dann nicht auslesen (invalid position). Insofern muss es hier ein anderer Datenpunkt sein.

Leider startet meine Kühlung via "manuell" oder "zeitgesteuert" aktuell gar nicht (kann allerdings an diesen ominösen 3 Tage Durchschnitt liegen).

Der einzige Weg meine Kühlung sicher zu starten ist via "Kühlung über mehrere Tage". Das funktioniert immer, nur kann ich das aktuell via Ebus weder steuern noch auslesen. Ich habe auch schonmal mittels "grab" die Befehle getracked, aber da stoße ich definitiv an meine Grenzen.

> Weiß irgendjemand wie das Äquivalent zu ManualCooling aus der 15.700.csv bei der 15.ctlv2.csv heisst oder heißen könnte? Vaillant wird doch nicht von Wärmepumpe zu Wärmepumpe das Interface ändern. Am 20. Juni 2024 20:27:28 MESZ schrieb mzwanzi ***@***.***>:

**Reply by Dette70** [reply]
So, ich habe nun für mich eine praktische Lösung:

Als erstes aktiviere ich "Kühlung über mehrere Tage" via App oder VR720. Danach greifen dei Einstellungen für den Datapoint "z1OpModeCooling", d.h. ich kann die Kühlung auf "manuell" oder "zeitgesteuert" stellen und die Zeiten funktionieren ebenso.

Warum man aber VORHER Kühlung über mehrere Tage aktivieren muss, erschließt sich mir nicht wirklich. Kann ich aber mit leben, wenn das o. g. Setup so bleibt und die Kühlung so steuerbar wird.

**Comment by dieterguenter** [comment]
So habe ich das aktuell auch realisiert. Stelle den Zeitraum am Ende der Heizperiode bis Mitte Oktober und kann  dann über z1opcoolingmode ein- und ausschalten. Allerdings filtert die Wärmepumpe den Kühlmodus nochmal und schaltet ihn nicht immer ein. Damit kann ich allerdings leben, solange der Kühlmodus verfügbar ist, wenn man ihn wirklich benötigt.

Am 23. Juni 2025 15:58:05 MESZ schrieb Dette70 ***@***.***>:

**Comment by mspbonn** [comment]
Hallo !

Hat das Schalten der Kühlung schon mal jemand komplett per `ebusctl` gesteuert ?

Mein Schema:
```
$ ebusctl info
version: ebusd 26.1.26.1
device: /dev/serial0, serial high speed, enhanced, firmware 1.1[8419].2[847f]
.
.
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0607;HW=5103"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=BASV2;SW=0562;HW=6304", loaded "vaillant/15.basv.csv"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0104;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master #8, ebusd
address 33: master #13
address 36: slave #8, ebusd, scanning
address 38: slave #13, scanned "MF=Vaillant;ID=V32;SW=0117;HW=9802", loaded "vaillant/38.v32.csv"
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0607;HW=5103", loaded "vaillant/76.vwzio.csv"
address ec: slave, scanned "MF=Vaillant;ID=SOL00;SW=0562;HW=6304", loaded "vaillant/ec.sol.sc.csv"

```

Der Wert von `Hc1CoolingEnabled` sagt offenbar nur, dass Kühlen überhaupt möglich ist. `Z1CoolingOpMode` ist bei mit `manual`.
Die oben erwähnten Werte um das "Kühlen für mehrere Tage" per `ebusctl` zu aktivieren, finde ich bei mir nicht. Den einzigen Zugriff, den ich auf die Anlage habe, ist das VRC 720f/2 und mein eBus-Adapter auf dem Raspberry Pi. Ich möchte gerne per ssh-Sitzung aus der Ferne das Kühlen remote aktivieren können.

Hat schon mal jemand mit `ebusctl grab` und `ebusctl grab result` versucht die Kommunikation zw. VRC 720f/2 und den anderen Devices beim Ein- bzw. Ausschalten der Kühlung mitzuschneiden und zu analysieren ?

Verwenden dieser Definition aus 15.700.csv
```
.
.
r,,,ManualCooling,Manual cooling,,,b524,020000000600,ign,,IGN:4,,,,,value,,UIN,,,,"number of days of manual cooling (max. 99, 0=off, set temp via z1CoolingTemp)"
w,,,ManualCooling,Manual cooling,,,b524,020100000600,value,,UIN,,,,"number of days of manual cooling (max. 99, 0=off, set temp via z1CoolingTemp)"
.
.

```
führt leider zu
```
$ ebusctl r -c basv ManualCooling
ERR: invalid position in decode: basv ManualCooling 
```

Passt also irgendwie nicht zu meiner Anlage.

**Reply by mf76130** [reply]
also bei mir geht das einfach mit 2 Scripts:
`alias: manuelles kühlen
sequence:
  - data:
      topic: ebusd/700/ManualCooling/set
      payload: "14"
    action: mqtt.publish
mode: single
description: ""
`

und dann zum ausschalten wieder auf 0:
`alias: manuelles kühlen ausschalten
sequence:
  - data:
      topic: ebusd/700/ManualCooling/set
      payload: "0"
    action: mqtt.publish
mode: single
description: ""
`

**Comment by mspbonn** [comment]
Das funktioniert bei meinem Schema nicht:

```
ebusctl r -m 0 -c basv ManualCooling
ERR: invalid position in decode

```
